modules/javafx.controls/src/test/java/test/com/sun/javafx/scene/control/behavior/AccordionBehaviorTest.java

```diff
@@ -35,7 +35,7 @@ public class AccordionBehaviorTest {
     // Panes added / removed, focus should move along with it
     // Pane gets focused, the focus model should reflect it
 
-    @Test public void focusGainedIsCaughtByBehavior() {
-
+    @Test
+    public void focusGainedIsCaughtByBehavior() {
```

Comment:
Wait, is this an empty test? Why?

Reply:
Intentionally did not want to change the number of tests, which should be double checked during the review.  Leaving as is for now.

Reply:
Agree.